packages/farms/src/v2/fetchFarmsV2.ts

```diff
@@ -131,18 +131,18 @@ export async function farmV2FetchFarms({
     decimals,
   )
 
-  const tokensWithoutPrice = farmsDataWithPrices.reduce<Map<string, TokenInfo>>((acc, cur) => {
+  const tokensWithoutPrice = farmsDataWithPrices.reduce<Map<string, CurrencyParams>>((acc, cur) => {
     if (cur.tokenPriceBusd === '0') {
       acc.set(cur.token.address, cur.token)
     }
     if (cur.quoteTokenPriceBusd === '0') {
       acc.set(cur.quoteToken.address, cur.quoteToken)
     }
     return acc
-  }, new Map<string, TokenInfo>())
+  }, new Map<string, CurrencyParams>())
   const tokenInfoList = Array.from(tokensWithoutPrice.values())
   if (tokenInfoList.length) {
-    const prices = await getCurrencyPricesByTokenInfoList(tokenInfoList)
+    const prices = await getCurrencyListUsdPrice(tokenInfoList)
```

Comment:
The prices here is assuming `Record<Address, number>`

Reply:
Will need to use `getCurrencyKey` to get the key of the price map

Reply:
good catch